src/components/Theme/src/default-theme.js

```diff
@@ -16,6 +16,16 @@ export default function defaultTheme() {
 			overlay: 'rgba(0, 0, 0, 0.3)',
 			primary: '#000000',
 		},
+		fonts: {
+			mobileBaseSize: '16px',
+			mobileSizeScale: 1.17,
+			mobileBaseLineHeight: '1.5em',
```

Comment:
line height units should be baseless, but I'm not sure how that impacts calculations.